scenarios/evaluate/AI_RedTeaming/AI_RedTeaming.ipynb

```diff
@@ -0,0 +1,447 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# AI Red Teaming Agent for Generative AI models and applications in Azure AI Foundry\n",
+    "\n",
+    "## Objective\n",
+    "This notebook walks through how to use Azure AI Evaluation's AI Red Teaming Agent functionality to assess the safety and resilience of AI systems against adversarial prompt attacks. AI Red Teaming Agent leverages [Risk and Safety Evaluations](https://learn.microsoft.com/en-us/azure/ai-foundry/concepts/evaluation-metrics-built-in?tabs=warning#risk-and-safety-evaluators) to help identify potential safety issues across different risk categories (violence, hate/unfairness, sexual content, self-harm) combined with attack strategies of varying complexity levels from [PyRIT](https://github.com/Azure/PyRIT), Microsoft AI Red Teaming team's open framework for automated AI red teaming.\n",
+    "\n",
+    "## Time\n",
+    "You should expect to spend about 30-45 minutes running this notebook. Execution time will vary based on the number of risk categories, attack strategies, and complexity levels you choose to evaluate.\n",
+    "\n",
+    "## Before you begin\n",
+    "\n",
+    "### Installation\n",
+    "Execute the following cells in a terminal window. Install the following packages required to execute this notebook, then use the virtual environment created as kernel to run the remainder of this notebook."
```

Comment:
Maybe don't have these in a cell because someone might just accidentally run these in the notebook, pull it out into a code block in markdown